debug.go

```diff
@@ -0,0 +1,347 @@
+package kvdb
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultLogLocation = "/var/lib/osd/kvdb_audit.log"
```

Comment:
can you change this to /var/cores since that's where we are storing all of your support bundles?